src/panels/lovelace/editor/config-elements/hui-area-card-editor.ts

```diff
@@ -65,7 +65,6 @@ export class HuiAreaCardEditor
 
   private _valueChanged(ev: CustomEvent): void {
     const config = ev.detail.value;
-    Object.keys(config).forEach((k) => config[k] === "" && delete config[k]);
```

Comment:
Dont we need this for the text fields?

Reply:
No? <https://github.com/home-assistant/frontend/pull/11754>?

Reply:
oh right